airflow/dag_processing/bundles/git.py

```diff
@@ -154,33 +153,23 @@ def __init__(
         self.git_conn_id = git_conn_id
         self.repo_url = repo_url
 
-        def log_debug(msg, **kwargs):
-            if not log.isEnabledFor(logging.DEBUG):
-                return
-            # ugly; replace when structlog implemented
-            context = dict(
-                bundle_name=self.name,
-                version=self.version,
-                bare_repo_path=self.bare_repo_path,
-                repo_path=self.repo_path,
-                versions_path=self.versions_dir,
-                git_conn_id=self.git_conn_id,
-                repo_url=self.repo_url,
-            )
-            context.update(kwargs)
-
-            for k, v in context.items():
-                msg += f" {k}='{v}'"
-            log.debug(msg)
-
-        self._log_debug = log_debug
-        log_debug("bundle configured")
+        self._log = log.bind(
```

Comment:
Why not just "self.log"?

Reply:
then it becomes "public" .... i mean, no one subclassing gitdagbundle should expect any interface backcompat, but, i guess i just try to err on side of explicitly private. just tend to be cautious about "public interface"